app/src/main/java/com/keylesspalace/tusky/adapter/PollAdapter.kt

```diff
@@ -0,0 +1,108 @@
+package com.keylesspalace.tusky.adapter
```

Comment:
Missing license/copyright (and PollViewData too)

Reply:
thx!